filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -127,6 +127,17 @@ Kafka fetch settings:
 *`max`*:: The maximum number of bytes to read per request. Defaults to 0
 (no limit).
 
+===== `expand_event_list_from_field`
+
+If the fileset using this input expects to receive multiple messages bundled under a specific field then the config option `expand_event_list_from_field` value can be assigned the name of the json group object.
```

Comment:
"the name of the json group object" -> "the name of the field" is I think clearer about what value should go in the config

"fielsets" -> "filesets" in the following line